packages/world/src/NatureLib.sol

```diff
@@ -91,6 +155,30 @@ library NatureLib {
     return result;
   }
 
+  function getLeafDrops(ObjectTypeId objectType, uint256 randomSeed)
+    internal
+    view
+    returns (ObjectAmount[] memory result)
+  {
+    uint256[] memory distribution = new uint256[](2);
+    distribution[0] = 43; // No sapling: 43%
+    distribution[1] = 57; // 1 sapling: 57%
```

Comment:
where do these numbers come from? if we want ~1 sapling per tree shouldn't the probability for sapling be much lower?

Reply:
wasn't sure about those, they get adjusted with the current supply of seeds (so 57% is the max). So should we have different distributions for different seeds depending on the max number of leaves for each one?

Reply:
If it's not too complex to do i'd say so yeah. Would probably also turn the max drop rate way down (closer to maybe like 2-3 saplings per tree) so that saplings are not too abundant even in the beginning and there is an incentive to mine all the leaves (to avoid floating leaves everywhere)